libcontainerd/container_windows.go

```diff
@@ -168,7 +170,8 @@ func (ctr *container) waitExit(pid uint32, processFriendlyName string, isFirstPr
 		}
 
 		if si.State == StateExit && ctr.restartManager != nil {
-			restart, wait, err := ctr.restartManager.ShouldRestart(uint32(exitCode), false)
+			ctr.exitedAt = time.Now()
```

Comment:
No need to keep this as a property if only used locally. `time.Since()` is shorter and more readable.